doc/cephadm/install.rst

```diff
@@ -0,0 +1,285 @@
+     host1# ssh-copy-id -f -i ceph.pub root@host3
+
+#. Tell Ceph that the new node is part of the cluster::
+
+     # ceph orch host add *newhost*
+
+   For example::
+
+     # ceph orch host add host2
+     # ceph orch host add host3
+
+
+Deploying additional monitors (optional)
+========================================
+
+Normally a Ceph cluster has three or five monitor daemons spread
+across different hosts.  As a rule of thumb, you should deploy five
+monitors if there are five or more nodes in your cluster.
+
+.. _CIDR: https://en.wikipedia.org/wiki/Classless_Inter-Domain_Routing#CIDR_notation
+
+When Ceph knows what IP subnet the monitors should use it can automatically
+deploy and scale monitors as the cluster grows (or contracts).  By default,
+Ceph assumes that other monitors should use the same subnet that the first
+monitor IP belongs to.
+
+If your Ceph monitors (or the entire cluster) live on a single subnet,
+then cephadm will automatically add up to give monitors as you add new
+hosts to the cluster, and no further steps are necessary.
+
+* If there is a specific IP subnet that should be used by monitors, you
+  can configure that in `CIDR`_ format (e.g., ``10.1.2.0/24``) with::
+
+    # ceph config set mon public_network *<mon-cidr-network>*
+
+  For example::
+
+    # ceph config set mon public_network 10.1.2.0/24
+
+  Cephadm will only deploy new monitor daemons on hosts that have IPs
+  configured in the configured subnet.
+
+* If you want fewer (or more) than the default of 5 monitors::
+
+    # ceph orch apply mon *<number-of-monitors>*
+
+* To deploy monitors on a specific set of hosts::
+
+    # ceph orch apply mon *<host1,host2,host3,...>*
+
+  Be sure to include the first (bootstrap) host in this list.
+
+* You can control which hosts the monitors run on by making use of
+  host labels.  To set the ``mon`` label to the appropriate
+  hosts::
+
+    # ceph orch host label add *<hostname>* mon
+
+  To view the current hosts and labels,::
+
+    # ceph orch host ls
+
+  For example::
+
+    # ceph orch host label add host1 mon
+    # ceph orch host label add host2 mon
+    # ceph orch host label add host3 mon
+    # ceph orch host ls
+    HOST   ADDR   LABELS  STATUS  
+    host1         mon             
+    host2         mon             
+    host3         mon             
+    host4
+    host5
+
+  Then tell cephadm to deploy monitors based on the label::
+
+    # ceph orch apply mon label:mon
+
+* You can explicitly specify the IP address or CIDR network for each monitor
+  and control where it is placed.  To disable automated monitor deployment::
+
+    # ceph orch apply mon --unmanaged
+
+  To deploy each additional monitor::
+
+    # ceph orch daemon add mon *<host1:ip-or-network1> [<host1:ip-or-network-2>...]*
+
+  For example, to deploy a second monitor on ``newhost1`` using an IP
+  address ``10.1.2.123`` and a third monitor on ``newhost2`` in
+  network ``10.1.2.0/24``,::
```

Comment:
@liewegas could you replace all ",::" with "::"? as ",::" will be translated into ",:" by sphinx.